lib/ts/supertokens.ts

```diff
@@ -49,6 +50,9 @@ export default class SuperTokens {
     telemetryEnabled: boolean;
 
     constructor(config: TypeInput) {
+        if (config.debug) {
```

Comment:
- we also want it to work for with the env var like before (backwards compatibility)
- Doing implicit if is not a good idea. Make it `if (config.debug === true)`